todo_app/data/classes.py

```diff
@@ -0,0 +1,18 @@
+class Item:

```

Comment:
Usually, classes go in a separate file with their name, i.e. "item.py" for the Item class